From 1acd307be20da130a9476dd00f4849b562571e27 Mon Sep 17 00:00:00 2001
From: Bai Ping <ping.bai@nxp.com>
Date: Mon, 10 Jul 2017 15:15:26 +0800
Subject: [PATCH 2091/5242] MLK-15935-01 ARM64: dts: update the critical trip
 point temp for 8qm/qxp

commit  004c29a4832d43032db889293bdded6e388c9907 from
https://source.codeaurora.org/external/imx/linux-imx.git

Update the default critical trip point temp to 127C to align with the
SCFW's panic alarm temp.

Signed-off-by: Bai Ping <ping.bai@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi  |   10 +++++-----
 arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi |    2 +-
 2 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
index a8a8828..68c9eb9 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
@@ -917,7 +917,7 @@
 					type = "passive";
 				};
 				cpu_crit0: trip1 {
-					temperature = <110000>;
+					temperature = <127000>;
 					hysteresis = <2000>;
 					type = "critical";
 				};
@@ -935,7 +935,7 @@
 					type = "passive";
 				};
 				cpu_crit1: trip1 {
-					temperature = <110000>;
+					temperature = <127000>;
 					hysteresis = <2000>;
 					type = "critical";
 				};
@@ -953,7 +953,7 @@
 					type = "passive";
 				};
 				gpu_crit0: trip1 {
-					temperature = <110000>;
+					temperature = <127000>;
 					hysteresis = <2000>;
 					type = "critical";
 				};
@@ -971,7 +971,7 @@
 					type = "passive";
 				};
 				gpu_crit1: trip1 {
-					temperature = <110000>;
+					temperature = <127000>;
 					hysteresis = <2000>;
 					type = "critical";
 				};
@@ -989,7 +989,7 @@
 					type = "passive";
 				};
 				drc_crit0: trip1 {
-					temperature = <110000>;
+					temperature = <127000>;
 					hysteresis = <2000>;
 					type = "critical";
 				};
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
index d81c690..de7f758 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
@@ -702,7 +702,7 @@
 					type = "passive";
 				};
 				cpu_crit0: trip1 {
-					temperature = <110000>;
+					temperature = <127000>;
 					hysteresis = <2000>;
 					type = "critical";
 				};
-- 
1.7.9.5

